package-testing/pre/000_install_package.rb

```diff
@@ -2,13 +2,24 @@
   workstation = find_at_most_one('workstation')
 
   step 'Install pdk package' do
+    if ENV['LOCAL_PKG']
+      pkg = File.basename(ENV['LOCAL_PKG'])
+      scp_to(workstation, ENV['LOCAL_PKG'], pkg)
```

Comment:
I don't think `basename` touches the filesystem. I was thinking of adding a `File.exist?` check but I wasn't sure how beaker expected a pre-suite to indicate failure, do you just `abort` or what?